src/server/cli/index.ts

```diff
@@ -6,8 +6,8 @@ import { unWrapArgs } from './parseArgs';
 
 export default function init(opts: CLI): void {
   if (!opts.help) {
-    const { ssh, server, command, ssl } = unWrapArgs(opts);
-    WeTTy(ssh, server, command, ssl).catch(err => {
+    const { ssh, server, client, command, ssl } = unWrapArgs(opts);
+    WeTTy(ssh, server, client, command, ssl).catch(err => {
```

Comment:
I believe this would be a breaking change to the api.
Either update the package.json to reflect this or make this arg optional. 

Please also update the docs aswell